### PR TITLE
EDU-1347 Streamline document input UI elements on document page

### PR DESCRIPTION
**Pull request by ahelmberger**
Closes https://educandu.atlassian.net/browse/EDU-1347

### CODE REVIEW

File: src/components/document-inputs-panel.yml [hunk 1]
```diff
@@ -15,5 +15,14 @@ documentInputsPanel:
...
     de: Es wurden noch keine Eingaben gesendet
   emptyStateSubtitle:
-    en: Fill in the document and click the submit button to send your input.
+    en: Fill in the document and click the submit button to submit your input.
     de: Fülle das Dokument aus und klicke auf den Senden-Button, um deine Eingabe abzuschicken.
+  emptyStateSubtitleDisabled:
+    en: For this document there are no previously submitted inputs, and submitting new inputs is currently disabled.
+    de: Für dieses Dokument liegen keine bisher gesendete Eingaben vor, und das Senden neuer Einagben ist derzeit deaktiviert.
+  inputSubmittingDisabledAlert:
+    en: Submitting inputs ist currently disabled for this document.
```
Comment (adrianaluchian) [inline src/components/document-inputs-panel.yml:24]: No it 'ist' not disabled 😋

**Review by adrianaluchian — APPROVED** [review]

File: src/components/document-inputs-panel.yml (hunk 1 above)
Comment (ahelmberger) [reply src/components/document-inputs-panel.yml]: Now it 'is' :-)

Comment (adrianaluchian) [reply src/components/document-inputs-panel.yml]: G(oo/u)(d/t) 😵

Comment (ahelmberger) [reply src/components/document-inputs-panel.yml]: :rofl:

**Pull request merged**